android/source/VideoLocker/src/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -66,7 +66,7 @@
     protected IStorage storage;
     protected ISegment segIO;
     protected boolean runOnTick = true;
-    protected final Logger logger = new Logger(getClass().getName());
+    protected transient final Logger logger = new Logger(getClass().getName());
```

Comment:
I'm not an expert, but as best I can tell, you only need ``transient`` in classes that implement ``Serializable`` which this one doesn't. Also, we shouldn't be using ``Serializable`` anyway.